src/router/routes/api/draftmancer/publish.ts

```diff
@@ -122,103 +79,25 @@ export const handler = async (req: Request, res: Response) => {
       players: [],
     };
 
+    let drafterName: string = '';
     for (const player of publishDraftBody.players) {
-      const draftmancerPicks: DraftmancerPick[] = [];
       let mainboard: number[][][] = setupPicks(2, 8);
       let sideboard: number[][][] = setupPicks(1, 8);
-      const pickorder: number[] = [];
-      const trashorder: number[] = [];
-
-      for (const pick of player.picks) {
-        // we are going to ignore burned cards
-        for (const index of pick.picks) {
-          const pack: number[] = pick.booster.map((oracleId) => upsertCardAndGetIndex(cards, oracleId));
-
-          const pickIndex = pack[index];
-
-          pickorder.push(pickIndex);
-          draftmancerPicks.push({
-            booster: pack,
-            pick: pickIndex,
-          });
-        }
-      }
-
-      for (const oracleId of player.decklist.main) {
```

Comment:
Before this these two loops over the main and sideboard would be done for bots, but those would be overwritten after doing the bot deckbuild().

As far as I can see anything in player.decklist.main and player.decklist.side should also be in the picks arrays, which are processed by getPicksFromPlayer below and adds to the cards array